components/grid/columns/menu.md

```diff
@@ -165,6 +250,8 @@ To hide a column from the Column Chooser set the `VisibleInColumnChooser` proper
 
 * If the Grid has a [frozen]({%slug grid-columns-frozen%}) column (`Locked="true"`), that column cannot be unfrozen from the column menu.
 
+* If you are using the [Column Chooser Template]({%slug grid-templates-column-chooser%}) or you are grouping the columns into [sections](#sections), it is recommended to add the `Title` parameter to all Grid Columns.
+
 ## See Also
 
-  * [Live Demo: Visible Columns](https://demos.telerik.com/blazor-ui/grid/columns)
+  * [Live Demo: Grid Custom Column Menu](https://demos.telerik.com/blazor-ui/grid/custom-column-menu)
```

Comment:
Consider linking both column menu demos here

Reply:
Great idea, I have no idea why I missed it